sampler_remote_test.go

```diff
@@ -405,7 +408,8 @@ func TestSamplerQueryError(t *testing.T) {
 
 	sampler.Close() // stop timer-based updates, we want to call them manually
 
-	sampler.UpdateSampler()
+	err := sampler.UpdateSampler()
+	assert.NotNil(t, err)
```

Comment:
`UpdateSampler` is called in other places. We should `assert.NoError` on those calls.